cartographer/cloud/client/map_builder_stub.cc

```diff
@@ -164,9 +164,12 @@ void MapBuilderStub::SerializeStateToFile(bool include_unfinished_submaps,
   ::grpc::Status status;
   async_grpc::Client<handlers::WriteStateToFileSignature> client(
       client_channel_);
-  CHECK(client.Write(request, &status))
-      << "code: " << status.error_code()
-      << " reason: " << status.error_message();
+  if (!client.Write(request, &status)) {
+    LOG(ERROR) << "WriteStateToFileRequest failed - "
```

Comment:
Removing this CHECK - e.g. an empty filename from a ROS service request could crash the client otherwise.